api/envoy/config/cluster/aggregate/cluster.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
```

Comment:
This needs to be in a versioned namespace, i.e. `envoy.config.cluster.aggregate.v2`

Reply:
Done